pkg/server/settingswatcher/settings_watcher.go

```diff
@@ -236,7 +238,7 @@ func (s *SettingsWatcher) handleKV(
 	name, val, tombstone, err := s.dec.DecodeRow(roachpb.KeyValue{
 		Key:   kv.Key,
 		Value: kv.Value,
-	})
```

Comment:
For this, I think you should allocate a single alloc for reuse.

Reply:
What I was saying here is that you could create a long-lived `alloc` because this is single-threaded. It doesn't really matter I guess.